src/cloudimagedirectory/transformer.py

```diff
@@ -80,6 +80,7 @@ def run(
             transform.TransformerIdxListImageLatestGoogle,
             transform.TransformerIdxListImageLatestAWS,
             transform.TransformerIdxListImageLatestAZURE,
+            transform.TransformerV2All,
```

Comment:
For some reason - as soon as I add this transformer - the google test starts to fail with the following message: 
```
―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― test_google_transformer_command ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
[gw3] linux -- Python 3.11.2 /home/miyunari/.cache/pypoetry/virtualenvs/cloudimagedirectory-qP5mipiD-py3.11/bin/python

runner = <click.testing.CliRunner object at 0x7fbec1214e90>
tmp_path = PosixPath('/tmp/pytest-of-miyunari/pytest-6/popen-gw3/test_google_transformer_comman0')

    def test_google_transformer_command(runner, tmp_path):
        """Verify that we can transform Google data."""
        runner.invoke(
            transformer.run,
            [
                "-f",
                "tests/transformer/testdata/input/raw/google/all.json",
                "-op=.",
                f"-dp={tmp_path}",
                "--filter.until=none",
            ],
        )
    
        # Ensure the directory was made.
>       assert os.path.isdir(f"{tmp_path}/v1/google/global")
E       AssertionError: assert False
E        +  where False = <function isdir at 0x7fbede203c40>('/tmp/pytest-of-miyunari/pytest-6/popen-gw3/test_google_transformer_comman0/v1/google/global')
E        +    where <function isdir at 0x7fbede203c40> = <module 'posixpath' (frozen)>.isdir
E        +      where <module 'posixpath' (frozen)> = os.path

tests/transformer/google_test.py:22: AssertionError

 tests/transformer/google_test.py ⨯                                                                                                             50% █████     
 tests/transformer/test_all.py ✓ 
```
Any idea what could cause this? Or how to debug this further? :/